device/bgp.yaml

```diff
@@ -0,0 +1,87 @@
+components:
+  schemas:
+    Device.Bgp:
+      x-include:
+      - '../common/common.yaml#/components/schemas/Named.Object'
+      x-status: under-review
+      description: >-
+        Container of basic emulated BGP peer settings.
+
+        This container has the following optional sub containers.
+        Device.BgpAdvanced, Device.BgpCapability, Device.BgpSrTePolicy
+        
+        This container has the following optional lists of containers.
+        Device.Bgpv4RouteRange, Device.Bgpv6RouteRange
+      type: object
+      properties:
+        active:
+          description: >-
+            Activates the protocol which means the protocol can be started
+            or stopped.
+          type: boolean
+          default: True
+        router_id:
+          description: >-
+            The IP address of the emulated router
```

Comment:
Q: Is the router_id really supposed to be the IP address of the emulated router?

Reply:
updated description